src/calendar/tests/CalendarHighlight.spec.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react/addons';
+import CalendarHighlight from '../CalendarHighlight';
+
+const TestUtils = React.addons.TestUtils;
+
+describe('The CalendarHighlight Component', function () {
+  beforeEach(function () {
+    this.spyCx = spyOn(CalendarHighlight.prototype.__reactAutoBindMap, 'cx').and.returnValue('my-class');
```

Comment:
Yes, I fear that. However, checking the class names has a number of issues as the bem syntax implies there will be a lot of them.

The other approach is to mock the BemMixin module instead of the cx function. It is not the simplest thing around though :)